### DISCUSSION TITLE
Do i need to remove App.js and App.css

### DISCUSSION THREAD

**Discussion by jakefroeb** [nss-day-cohort-46/How-to-Ask-for-Help#54] [General]
**_Ticket MUST be completed before getting help from an instructor_**

#### Have you searched the other issue tickets? If not, do that first. 
- [ x] Yes

####  REQUIRED Add Tags to Issue Ticket and Assign to Project Board
 

- [ x] Added label to specific project/assignment that you need help with
- [x ] Added to `c46 Help-Thread` project board

#### REQUIRED Describe your issue:
>> when creating a new react app do i always need to remove the App.js and App.css

#### REQUIRED What have you googled? **_(You must have at least 3 links related to your issue)_**

- [ ] 1. https://create-react-app.dev/docs/getting-started/
- [ ] 2. https://www.quora.com/After-I-created-a-project-using-a-create-react-app-how-can-I-delete-it-from-my-computer
- [ ] 3. https://dev.to/erdn0/what-to-delete-after-creating-react-app-10d8

**Comment by jakefroeb** [comment]
seems the answer is yes i just really want some verbal verification from an instructor

**Comment by jisie** [comment]
You don't have to. But that's the placeholder component that `create-react-app` provides. So if you still want to call your outer level component `App`, you can keep it. You can also rename the files.
